homeassistant/components/airzone/__init__.py

```diff
@@ -0,0 +1,66 @@
+"""The Airzone integration."""
+from __future__ import annotations
+
+from aioairzone.common import ConnectionOptions
+from aioairzone.localapi_device import AirzoneLocalApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import AirzoneUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+class AirzoneDevice(CoordinatorEntity):
+    """Define an Airzone device."""
```

Comment:
```suggestion
    """Define an Airzone entity."""
```

Reply:
Done in 9a96604fc814869ecb594340ace7a1325230b05b